src/NServiceBus.Core/ServicePlatform/Retries/ManualRetryNotificationBehavior.cs

```diff
@@ -0,0 +1,65 @@
+﻿namespace NServiceBus
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using Pipeline;
+    using Routing;
+    using Transport;
+
+    class ManualRetryNotificationBehavior : IForkConnector<ITransportReceiveContext, ITransportReceiveContext, IRoutingContext>
+    {
+        internal const string RetryUniqueMessageIdHeaderKey = "ServiceControl.Retry.UniqueMessageId";
+        internal const string RetryConfirmationQueueHeaderKey = "ServiceControl.Retry.AcknowledgementQueue";
+
+        public async Task Invoke(ITransportReceiveContext context, Func<ITransportReceiveContext, Task> next)
+        {
+            var useRetryAcknowledgement = IsRetriedMessage(context, out var id, out var acknowledgementQueue);
+
+            if (useRetryAcknowledgement)
+            {
+                // notify the ServiceControl audit instance that the retry has already been acknowledged by the endpoint
+                context.Extensions.Set(new MarkAsAcknowledgedBehavior.State());
+            }
+
+            await next(context).ConfigureAwait(false);
+
+            if (useRetryAcknowledgement)
+            {
+                await ConfirmSuccessfulRetry().ConfigureAwait(false);
+            }
+
+            async Task ConfirmSuccessfulRetry()
+            {
+                var messageToDispatch = new OutgoingMessage(
+                    CombGuid.Generate().ToString(),
+                    new Dictionary<string, string>
+                    {
+                        { "ServiceControl.Retry.Successful", DateTimeOffset.UtcNow.ToString("O") },
```

Comment:
Should we use `ToWireFormattedString` here?

Reply:
hm, `ToWireFormattedString` seems to use a slightly different format, "O" uses ISO8601. But the docs states that message headers contain the format defined in ToWireFormattedString (https://docs.particular.net/nservicebus/messaging/headers#timestamp-format) so I guess that might be better.

Reply:
this will require SC to use the same Helper to be able to correctly deserialize the string value, compared to just using DateTimeOffset.Parse, is that something we want to do?

Reply:
Actually, SC just puts "true" there when the audit instance sends the message and the main instance cares only about the existence of the header. I am fine with changing the audit to use the wire formatted string

Reply:
we could technically set the same header as we set for the audit instance. The idea was that this might be interesting information for SP/SI as we currently show timestamp of retry attempts but not the timestamp of the successful retry.